src/components/OptionRow.js

```diff
@@ -100,7 +100,13 @@ class OptionRow extends Component {
             || this.props.option.text !== nextProps.option.text
             || this.props.option.alternateText !== nextProps.option.alternateText
             || this.props.option.descriptiveText !== nextProps.option.descriptiveText
-            || this.props.option.brickRoadIndicator !== nextProps.option.brickRoadIndicator;
+            || this.props.option.brickRoadIndicator !== nextProps.option.brickRoadIndicator
+            || this.props.option.shouldShowSubscript !== nextProps.option.shouldShowSubscript
+            || this.props.option.icons !== nextProps.option.icons
+            || this.props.option.ownerEmail !== nextProps.option.ownerEmail
+            || this.props.option.subtitle !== nextProps.option.subtitle
+            || this.props.option.pendingAction !== nextProps.option.pendingAction
+            || this.props.shouldHaveOptionSeparator !== nextProps.shouldHaveOptionSeparator;
```

Comment:
Move `props.shouldHaveOptionSeparator` check to be after `props.isSelected` since it's more likely to change compared to others (early return optimisation)